boards.txt

```diff
@@ -439,6 +439,48 @@ nRF51Dongle.menu.softdevice.s130.upload.maximum_size=151552
 nRF51Dongle.menu.softdevice.s130.build.extra_flags=-DNRF51 -DS130 -DNRF51_S130
 nRF51Dongle.menu.softdevice.s130.build.ldscript=armgcc_s130_nrf51822_{build.chip}.ld
 
+
+TinyBLE.name= TinyBLE
+
+TinyBLE.upload.tool=sandeepmistry:openocd
+TinyBLE.upload.protocol=cmsis-dap
```

Comment:
I would argue our default selection has never worked, everyone has manually selected up until now, so it doesnt depend on it, but rather will be useful when 114 lands.